mcs/class/System/System.Net/HttpWebRequest.cs

```diff
@@ -393,23 +394,15 @@ void SetDateHeaderHelper (string headerName, DateTime dateTime)
 #if !MOBILE
 		[MonoTODO]
```

Comment:
Remove [MonoTODO]

Reply:
We don't actually do anything with the value - shouldn't we keep the attribute until it's actually implemented?